src/utils/transactions/parseTransactionAfterSigning.ts

```diff
@@ -1,24 +1,24 @@
-import {
-  Transaction,
-  TransactionOptions,
-  TransactionVersion
-} from '@elrondnetwork/erdjs';
+import { PlainSignedTransaction } from '@elrondnetwork/erdjs-web-wallet-provider/out/plainSignedTransaction';
+import { Transaction } from '@elrondnetwork/erdjs/out';
 import { TransactionServerStatusesEnum } from 'types/enums';
+import { newTransaction } from '../../models';
+import { SignedTransactionType } from '../../types';
 
 export function parseTransactionAfterSigning(
-  transaction: Transaction,
-  isLedger = false
+  signedTransaction: Transaction | PlainSignedTransaction
 ) {
-  // TODO: REMOVE
```

Comment:
@StanislavSava I removed the `isLedger` flag because `getTransactionsFromWalletUrl` metho already returns `version` and `options` properties on the plain signed transaction object